doc/vim-go.txt

```diff
@@ -629,6 +668,19 @@ if			      "inside a function", select contents of a function,
 
 
 ===============================================================================
+FUNCTIONS                                                       *go-functions*
+
+There is only one function at the moment and it is neovim only.
```

Comment:
We also have `go#complete#GetInfo()` which returns the description of the identifier under the cursor.  It can be also plugged into statusline. This function is also used for the `go_auto_type_info` feature.